### PR TITLE
Make the redis connection threshold a variable for alerting

### PR DESCRIPTION
**Pull request by ioanav104**
Allow mixin users more flexibility to set the threshold

### CODE REVIEW

**Comment by oliver006** [comment]
Thanks for the PR! The linter complained here: 
https://cloud.drone.io/oliver006/redis_exporter/1442/1/16

Can you make sense of that error? I'm not really familiar with the mixin stuff...

**Comment by ioanav104** [comment]
I'm having a look

File: contrib/redis-mixin/alerts/redis.libsonnet [hunk 1]
```diff
@@ -30,7 +30,7 @@
           },
           {
             alert: 'RedisTooManyConnections',
-            expr: 'redis_connected_clients > 100',
+            expr: 'redis_connected_clients > %(redisConnectionsThreshold)' % $._config,
```
Comment (kragniz) [inline contrib/redis-mixin/alerts/redis.libsonnet:33]:
```suggestion
            expr: 'redis_connected_clients > %(redisConnectionsThreshold)s' % $._config,
```

Comment (kragniz) [reply contrib/redis-mixin/alerts/redis.libsonnet:33]: jsonnet uses the same mini-language as python for string formatting, so you need to specify it's a string

https://docs.python.org/2/library/stdtypes.html#string-formatting

**Comment by oliver006** [comment]
Thanks for getting this sorted out!

**Pull request merged**